### PR TITLE
Modified samples to support ONNX models

### PR DESCRIPTION
**Pull request by mryzhov**
- Removed manual loading of IR .bin files (hello_classification, object_detection_sample_ssd)
- Corrected comments
- Added Note in README files

### CODE REVIEW

**Comment by ilyachur** [comment]
Please do not merge while we don't have test report.

**Comment by ilyachur** [comment]
@mryzhov Did you add tests for these samples?

**Comment by mryzhov** [comment]
> @mryzhov Did you add tests for these samples?

Just added, waiting for the test results.

File: inference-engine/samples/speech_sample/main.cpp [hunk 1]
```diff
@@ -567,7 +567,7 @@ int main(int argc, char *argv[]) {
         std::cout << ie.GetVersions(deviceStr) << std::endl;
         // -----------------------------------------------------------------------------------------------------
 
-        // --------------------------- 2. Read IR Generated by ModelOptimizer (.xml and .bin files) ------------
+        // --------------------------- 2. Read model from IR (.xml and .bin files) or ONNX (.onnx file) format -
```
Comment (aalborov) [inline inference-engine/samples/speech_sample/main.cpp:570]: I'd change this to ```Read a model in OpenVINO Intermediate Representation (.xml and .bin files) or ONNX (.onnx file) format.```

File: inference-engine/samples/benchmark_app/README.md [hunk 2]
```diff
@@ -116,6 +116,8 @@ If a model has mixed input types, input folder should contain all required files
 To run the tool, you can use public or Intel's pre-trained models. To download the models, use the OpenVINO [Model Downloader](@ref omz_tools_downloader_README) or go to [https://download.01.org/opencv/](https://download.01.org/opencv/).
 
 > **NOTE**: Before running the tool with a trained model, make sure the model is converted to the Inference Engine format (\*.xml + \*.bin) using the [Model Optimizer tool](../../../docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md).
+> 
+> The sample accepts models in ONNX format (.onnx) that don't require preprocessing.
```
Comment (aalborov) [inline inference-engine/samples/benchmark_app/README.md:120]: ```do not```, not ```don't```. We don't use contractions in documentation.

**Review by aalborov — CHANGES_REQUESTED** [review]
A couple of fixes needed, everything else lgtm.

**Comment by ilyachur** [comment]
@mryzhov Please fix review comments and update this PR when tests will be ready. I think we need to merge this PR for the current release.

File: inference-engine/samples/benchmark_app/README.md (hunk 2 above)
Comment (mryzhov) [reply inference-engine/samples/benchmark_app/README.md]: fixed

File: inference-engine/samples/speech_sample/main.cpp (hunk 1 above)
Comment (mryzhov) [reply inference-engine/samples/speech_sample/main.cpp]: fixed

**Comment by mryzhov** [comment]
> @mryzhov Please fix review comments and update this PR when tests will be ready. I think we need to merge this PR for the current release.

The new tests have been added and passed, I'll include the results to the ticket.

**Comment by ilyachur** [comment]
@mryzhov Please fix CI and I will merge this PR.

**Comment by ilyachur** [comment]
@aalborov Please take a look one more time.

**Review by aalborov — APPROVED** [review]
lgtm

**Pull request merged**